services/submission/pom.xml

```diff
@@ -67,6 +67,12 @@
       <groupId>com.google.protobuf</groupId>
       <artifactId>protobuf-java-util</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java-util</artifactId>
+      <version>3.12.4</version>
+      <scope>test</scope>
```

Comment:
please remove <version> + <scope> as it is already defined in the managed dependency section of the root-pom

Reply:
done